QuoteEngine.py

```diff
@@ -0,0 +1,30 @@
+import requests
+import yaml
+import random
+
+
+class QuoteEngine:
+
+    def __init__(self, logger):
+        self.quote_url = "https://cdn.rawgit.com/tulul/tulul-quotes/master/quote.yaml"  # noqa
+        # Note: rawgit does not have 100% uptime, but at
+        # least they're not throttling us.
+
+        self.logger = logger
```

Comment:
This is not used. Remove this.

If you want to use logging here, use Python's logging module.